cmd/pubsub/receive_adapter/main.go

```diff
@@ -53,14 +53,15 @@ func main() {
 		}
 	}
 
-	logger, _ := logging.NewLoggerFromConfig(loggingConfig, component)
+	sl, _ := logging.NewLoggerFromConfig(loggingConfig, component)
+	logger := sl.Desugar()
 	defer flush(logger)
-	ctx := logging.WithLogger(signals.NewContext(), logger)
+	ctx := logging.WithLogger(signals.NewContext(), logger.Sugar())
```

Comment:
Why not pass in `sl` here?

Reply:
Done.

My reasoning was that I wanted to pretend that `sl` didn't exist. But it's not a good reason, so switched.